pkg/kubectl/cmd/config/use_context.go

```diff
@@ -45,7 +45,7 @@ func NewCmdConfigUseContext(out io.Writer, configAccess clientcmd.ConfigAccess)
 	options := &useContextOptions{configAccess: configAccess}
 
 	cmd := &cobra.Command{
-		Use:     "use-context CONTEXT_NAME",
+		Use:     "use CONTEXT_NAME",
```

Comment:
I don't think we can change this now. That issue is years ago. We have other **"-context"** subcommands as well.  

@mengqiy @pwittrock PTAL.

Reply:
we don't have use-cluster command. this will enhance usability.

Reply:
It's a backwards incompatible change. We can't modify it.

Reply:
yes, i modify it to ``` Aliases: []string{"use"}```.
actually i want 
```
kubectl config use-context --> kubectl config use
kubectl config current-context --> kubectl config get
kubectl config get-contexts --> kubectl config list
```
i create a issue for this